include/libdnf/module/module_dependency.hpp

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef LIBDNF_MODULE_MODULE_DEPENDENCY_HPP
+#define LIBDNF_MODULE_MODULE_DEPENDENCY_HPP
+
+#include <string>
+#include <vector>
+
+namespace libdnf::module {
+
+
+class ModuleDependency {
+public:
+    ModuleDependency(std::string module_name, std::vector<std::string> streams)
```

Comment:
Small issue
`ModuleDependency(std::string module_name, std::vector<std::string> streams)`
vs
ModuleDependency(std::string & module_name, std::vector<std::string> & streams)
`std::string module_name` pass a copy of `module_name` to constructor => redundant copy

Reply:
What about to also add a move constructor?
`ModuleDependency(std::string && module_name, std::vector<std::string> && streams)`

Reply:
Nice

Reply:
I am only not sure if it's used correctly here: https://github.com/rpm-software-management/dnf5/pull/17/files#diff-e1c5753bcfca8e49721f0b42e243d88368343645cf917ed472164466592b0740R214